[2025-03-02T14:45:03.292+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_db_dag.save_raw_station_details_information scheduled__2025-03-02T14:30:00+00:00 [queued]>
[2025-03-02T14:45:03.298+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_db_dag.save_raw_station_details_information scheduled__2025-03-02T14:30:00+00:00 [queued]>
[2025-03-02T14:45:03.299+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-03-02T14:45:03.307+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): save_raw_station_details_information> on 2025-03-02 14:30:00+00:00
[2025-03-02T14:45:03.311+0000] {standard_task_runner.py:57} INFO - Started process 3260 to run task
[2025-03-02T14:45:03.315+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'api_to_db_dag', 'save_raw_station_details_information', 'scheduled__2025-03-02T14:30:00+00:00', '--job-id', '590', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpih0yh0hd']
[2025-03-02T14:45:03.319+0000] {standard_task_runner.py:85} INFO - Job 590: Subtask save_raw_station_details_information
[2025-03-02T14:45:03.366+0000] {task_command.py:410} INFO - Running <TaskInstance: api_to_db_dag.save_raw_station_details_information scheduled__2025-03-02T14:30:00+00:00 [running]> on host 684191f110a3
[2025-03-02T14:45:03.440+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_to_db_dag' AIRFLOW_CTX_TASK_ID='save_raw_station_details_information' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T14:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-02T14:30:00+00:00'
[2025-03-02T14:45:03.509+0000] {datalake_saver.py:45} INFO - Data saved successfully to /mnt/data/raw/velib_api/station_information/20250302/202503021445.json
[2025-03-02T14:45:03.509+0000] {python.py:183} INFO - Done. Returned value was: /mnt/data/raw/velib_api/station_information/20250302/202503021445.json
[2025-03-02T14:45:03.527+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=api_to_db_dag, task_id=save_raw_station_details_information, execution_date=20250302T143000, start_date=20250302T144503, end_date=20250302T144503
[2025-03-02T14:45:03.577+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-03-02T14:45:03.603+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
